src/prx/main.py

```diff
@@ -78,16 +78,16 @@ def write_csv_file(
     # write header
     with open(output_file, "w", encoding="utf-8") as file:
         file.write(f"# {json.dumps(prx_header)}\n")
-    flat_records["elevation_deg"] = np.rad2deg(flat_records.elevation_rad.to_numpy())
-    flat_records["azimuth_deg"] = np.rad2deg(flat_records.azimuth_rad.to_numpy())
+    flat_records["sat_elevation_deg"] = np.rad2deg(flat_records.elevation_rad.to_numpy())
+    flat_records["sat_azim_deg"] = np.rad2deg(flat_records.azimuth_rad.to_numpy())
```

Comment:
nit: suggest `sat_azim_deg` here to be consistent with `sat_elevation_deg` - or change to `sat_elev_deg` :)

Reply:
I'd favor `sat_azimuth_deg`